test/users.js

```diff
@@ -0,0 +1,159 @@
+const supertest = require("supertest");
+const assert = require("assert");
+const {models} = require("../sequelize");
+const app = require("../express/routes");
+
+//routes tests
+describe("GET /", () => {
+    it("should have status code 200", (done) => {
+        supertest(app)
+            .get("/")
+            .expect(200)
+            .end((err, res) => {
+                if (err) done(err);
+                done();
+            });
+    });
+});
+
+describe("GET /users", () => {
+    it("should return all users and have status code 200", (done) => {
+      supertest(app)
+            .get("/users")
+            .expect(200)
+            .end((err, res) => {
+                if (err) done(err);
+                done();
+            });
+    });
+});
+
+describe("GET /users/:id", () => {
+    it("should return user and have status code 200", async () => {
+        const testUser = await models.user.findOne();
+
+        const res = supertest(app)
+            .get(`/users/${testUser.id}`)
+            .expect(200);
```

Comment:
we can check user properties here, check the request has returned the user

Reply:
Left unchanged!